src/app/shared/loggerHelper.js

```diff
@@ -38,4 +39,24 @@ module.exports = {
     }
     req.log.info({ message, level: "INFO", requestId: req.id });
   },
+  getMiddlewareErrorHandlerMessage: (err, description) => {
+    const { config, request, response, ...desiredErrorProperties } = err;
+
+    const requestDataString = config?.data;
+
+    const credentialIssuerId =
+      requestDataString && JSON.parse(requestDataString)?.credentialIssuerId;
+
+    const message = {
+      err: desiredErrorProperties,
+      response: err?.response?.data,
+      description: description,
```

Comment:
I'm not sure how useful any of `err` is in the context of a core-back error response
```
err: {
       code: ERR_BAD_REQUEST
       config: {}
       message: Request failed with status code 400
       name: AxiosError
       stack: AxiosError: Request failed with status code 400
    at settle (/app/node_modules/axios/dist/node/axios.cjs:1967:12)
    at IncomingMessage.handleStreamEnd (/app/node_modules/axios/dist/node/axios.cjs:3066:11)
    at /opt/dynatrace/oneagent/agent/bin/1.283.132.20240205-143805/any/nodejs/nodejsagent.js:2167:48
    at t.runInContext (/opt/dynatrace/oneagent/agent/bin/1.283.132.20240205-143805/any/nodejs/nodejsagent.js:2278:24)
    at IncomingMessage.handleStreamEnd (/opt/dynatrace/oneagent/agent/bin/1.283.132.20240205-143805/any/nodejs/nodejsagent.js:2167:18)
    at IncomingMessage.emit (node:events:525:35)
    at endReadableNT (node:internal/streams/readable:1359:12)
    at /opt/dynatrace/oneagent/agent/bin/1.283.132.20240205-143805/any/nodejs/nodejsagent.js:2167:48
    at t.runInContext (/opt/dynatrace/oneagent/agent/bin/1.283.132.20240205-143805/any/nodejs/nodejsagent.js:2278:24)
    at endReadableNT (/opt/dynatrace/oneagent/agent/bin/1.283.132.20240205-143805/any/nodejs/nodejsagent.js:2167:18)
    at Axios.request (/app/node_modules/axios/dist/node/axios.cjs:3877:41)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async sendParamsToAPI (/app/src/app/credential-issuer/middleware.js:30:27)
       status: 400
     }
```
I know people on call have seen this and based on the stack trace thought there was a dynatrace related issue 😅 

The response actually tells us what went wrong
```
response: {
       code: 1030
       journey: /journey/error
       message: Missing OAuth state in callback request
       statusCode: 400
     }
```

This certainly helps but as the ticket suggests maybe we could make core-back response logging part of the `coreBackService`?